roles/odf_setup/README.md

```diff
@@ -30,6 +30,8 @@ Role Variables
 | local_storage_devices            |                               | List         | No          | For LSO, a list of local devices that will be use as backend             |
 | ocs_default_storage_class        | storagecluster-cephfs         | String       | No          | Default storage class name                                               |
 | gatherer_image                   | registry.access.redhat.com/ubi8/ubi | String | No          | Image for disk-gatherer deployment                                       |
+| odf_oc_tool_path                   | `{{ oc_tool_path | default('/usr/local/bin/oc) }}` | String | No          | Path to the OpenShift Command Line Interface binary.
```

Comment:
This role was imported when we did not follow a naming convention on role variables, but now we do validate that on changes, so let's rename the variable to `odf_setup_oc_tool_path` or `os_oc_tool_path` that should satisfy the naming convention

I suggest to make this default to `/usr/local/bin/oc`, the override will come through the variable itself (`odf_setup_oc_tool_path` or the other var name)